### PR TITLE
fix: update region list for AMW location in ARM, Bicep and Policy

### PR DESCRIPTION
**Pull request by bragi92**
Result from RP:
image

### CODE REVIEW

**Review by Sohamdg081992 — DISMISSED** [review]

**Review by vishiy — APPROVED** [review]

File: AddonArmTemplate/FullAzureMonitorMetricsProfile.json [hunk 1]
```diff
@@ -10,14 +10,24 @@
       "defaultValue": "",
       "allowedValues": [
         "eastus2euap",
-        "centraluseuap",
-        "centralus",
+        "northcentralus",
```
Comment (vishiy) [inline AddonArmTemplate/FullAzureMonitorMetricsProfile.json:13]: Kaveesh - should we also be adding fairfax region (virginia) here, as same templates will be used there too..

Comment (bragi92) [reply AddonArmTemplate/FullAzureMonitorMetricsProfile.json:13]: Makes sense, I'll update the regions to include virginia.

Comment (bragi92) [reply AddonArmTemplate/FullAzureMonitorMetricsProfile.json:13]: updated

Comment (bragi92) [reply AddonArmTemplate/FullAzureMonitorMetricsProfile.json:13]: @vishiy Can you please approve the PR again when you get a chance. It should be good to merge in now.

**Review by vishiy — APPROVED** [review]

**Pull request merged**